.circleci/config.yml

```diff
@@ -1,50 +1,134 @@
 version: 2.1
 
 executors:
-  cif_executor:
+  build-executor:
     docker:
-      - image: circleci/openjdk:8-jdk
+      - image: circleci/openjdk:11-stretch-node
+    working_directory: /home/circleci/build
 
 jobs:
   build:
-    executor: cif_executor
+    executor: build-executor
     steps:
       - checkout
+
       - restore_cache:
           keys: 
             - maven-repo-v1-{{ checksum "pom.xml" }}
             - maven-repo-v1-
+
       - run:
           name: Build
-          command: |
-            java -version
-            mvn -v
-            mvn -B clean install -s .circleci/settings.xml
+          command: node .circleci/ci/build.js
+
       - save_cache:
           paths:
-            - ~/.m2
+            - .m2
           key: maven-repo-v1-{{ checksum "pom.xml" }}
-      - run:
-          name: Collect and save all test results
-          command: |
-            mkdir -p ~/test-results/junit/
-            find . -type f -regex ".*/target/surefire-reports/.*xml" -exec cp {} ~/test-results/junit/ \;
+
+      - persist_to_workspace:
+          root: /home/circleci/build
+          paths:
+            - configuration.json
+            - bundles/*/target/*.jar
+            - content/*/target/*.zip
+            - it/content/target/*.zip
+
       - store_test_results:
-          path: ~/test-results
+          path: test-results
+
       - store_artifacts:
-          path: ~/test-results/junit
+          path: test-results/junit
+
       - run:
           name: Upload Code Coverage
           command: bash <(curl -s https://codecov.io/bash) -F unittests
-          
+
+
+  integration-test-652:
```

Comment:
One comment somehow didn't get saved: you could create a `command` here, and make it configurable with the docker images. This way you can remove the duplicate code for `integration-test-652` and `integration-test-646` and have one single command with parameters for the docker images.

Reply:
I updated the PR with the YAML anchor syntax. Hope this looks better now.

Reply:
Nice. It's now a matter of taste, but I'd remove the extra empty lines between "blocks" of content, because I find it a bit difficult to read it and find out what exactly belongs to what, also in terms of indentation ... it's up to you, but I think it would help to clearly see the "blocks" without empty lines.

Reply:
@cjelger I removed the empty lines 👍